pkg/storage/couchdb/couchdbstore.go

```diff
@@ -141,11 +143,28 @@ func (c *CouchDBStore) Put(k string, v []byte) error {
 		valueToPut = wrapTextAsCouchDBAttachment(v)
 	}
 
-	_, err := c.db.Put(context.Background(), k, valueToPut)
+	if c.revIDs[k] != "" {
+		var m map[string]interface{}
+		if err := json.Unmarshal(valueToPut, &m); err != nil {
+			return fmt.Errorf("failed to unmarshal put value: %w", err)
+		}
+
+		m["_rev"] = c.revIDs[k]
```

Comment:
Shouldn't this increment the revision number? Or change it somehow?

Reply:
no